ext/opentelemetry-ext-http-requests/src/opentelemetry/ext/http_requests/__init__.py

```diff
@@ -119,18 +126,62 @@ def instrumented_request(self, method, url, *args, **kwargs):
     # different, then push the current URL, pop it afterwards)
 
 
-def disable():
+def _uninstrument():
+    # pylint: disable=global-statement
     """Disables instrumentation of :code:`requests` through this module.
 
     Note that this only works if no other module also patches requests."""
+    global _tracer
 
     if getattr(Session.request, "opentelemetry_ext_requests_applied", False):
         original = Session.request.__wrapped__  # pylint:disable=no-member
         Session.request = original
-
-
-def disable_session(session):
-    """Disables instrumentation on the session object."""
-    if getattr(session.request, "opentelemetry_ext_requests_applied", False):
-        original = session.request.__wrapped__  # pylint:disable=no-member
-        session.request = types.MethodType(original, session)
+        _tracer = None
+
+
+def _http_status_to_canonical_code(code: int, allow_redirect: bool = True):
```

Comment:
should we follow up and move this somewhere up the packages? this is duplicated in 4-5 places now.

I hesitate to suggest the api package, which is the only common dependency. Or maybe an opentelemetry-instrumentor-utils?

Reply:
I think we should move that to a new package in the future, I'd like to avoid doing that on this PR as I want this to be in asap. By the way, I opened an issue to initiate the discussion about that https://github.com/open-telemetry/opentelemetry-python/issues/610, I think an utils package for instrumentors is the best idea.